src/natcap/invest/utils.py

```diff
@@ -796,3 +793,50 @@ def matches_format_string(test_string, format_string):
     if re.fullmatch(pattern, test_string):
         return True
     return False
+
+
+def copy_spatial_files(spatial_filepath, target_dir):
```

Comment:
I didn't notice any test being moved for this function from `test_datastack` to `test_utils`, did we not have one before? Should we have one now?

Reply:
I don't think we have a unit test for this, which might be fine. If we want one, I suggest we add it on `main` rather than in this PR.

Reply:
Correct, there wasn't a test before.

Reply:
I think since we are not changing any functionality here, a test is no more necessary than it was before

Reply:
Sounds good to me!